### DISCUSSION TITLE
Updating values from the server

### DISCUSSION THREAD

**Discussion by kenteramin** [edmundhung/conform#717] [Q&A]
I have a company form, when you fill in your org number I want to pre-fill the form with data from a registry.
So I thought I could submit my form with some 'operation' and return the new initial values from the BE.
I can surely just modify reply object, but I wonder if there's a better way? It would be great to have an option on `.reply` to manually update some fields

**Comment by kenteramin** [comment]
Huh, actually it doesn't work this way, cuz setting the initial values won't update an existing form

**Reply by marcomuser** [reply]
I think you could force it to update by changing the formid.

**Reply by kenteramin** [reply]
What I'm doing atm basically, I imitate the update response. I wonder if it is alright :))
```
 intent: {
  type: "update" as const,
  payload: {
    name: "company",
    value: details,
   },
},
```
